data/mods/Arcana/monsters.json

```diff
@@ -1987,5 +1987,33 @@
     "special_attacks":[["FLAMETHROWER", 10]],
     "description":"A massive animated conflagration, crackling like burning logs as it moves.",
     "flags":["NOHEAD", "HARDTOSHOOT", "WEBWALK", "FLIES", "FIREY", "WARM", "FIREPROOF", "SEES"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_moruboru",
```

Comment:
I like this little guy. I wonder if we could fit it in with the extant mainline triffids?